src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -113,7 +113,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $kernel = $this->getContainer()->get('kernel');
 
         // Define Root Path to App folder
-        $transPaths = array($kernel->getRootDir().'/Resources/');
+        $transPaths = array(
+            $kernel->getRootDir().'/Resources/',
+            $kernel->getProjectDir().'/',
```

Comment:
There is one problem here. These paths are used to generate `views` and `translations` folders paths (see L275 & L295). With Flex, `views` is renamed by `templates`, so translations could be extracted but translation keys used in templates won't be extracted.

https://github.com/symfony/symfony/issues/24807#issuecomment-341669603

Reply:
I propose you a simple solution in PR.